src/panels/edit-panel/panelCtrl.ts

```diff
@@ -84,11 +83,23 @@ export class PlotlyPanelCtrl extends MetricsPanelCtrl {
   async onUpload(net: any) {
     this.panel.predictionSettings.json = JSON.stringify(net);
     this.publishAppEvent(AppEvents.alertSuccess, ['File Json Caricato']);
+    this.panel.predictionSettings.predittori = net.Predictors.map((a, index) => {
```

Comment:
In inglese come il resto sarebbe meglio, quindi non "predittori" ma "predictors"